src/config.ts

```diff
@@ -4,16 +4,17 @@ import MainScene from './scenes/mainScene';
 export default {
   type: Phaser.AUTO,
   parent: 'game',
-  backgroundColor: '#33A5E7',
+  backgroundColor: '#ADD8E6',
   scale: {
     // has to be aspect ratio of 2:1
     width: 1000,
     height: 500,
   },
   physics: {
-    default: 'arcade',
-    arcade: {
-      gravity: { y: 200 },
+    default: 'matter',
+    matter: {
+      gravity: { y: 1 },
+      // debug: true,
```

Comment:
flip this on and off to see bounding boxes for collisions, nice for debugging

Reply:
![image](https://user-images.githubusercontent.com/60610467/233865807-f2c0b0ab-359c-4c1c-8241-d732d4813ba8.png)
Is this an easy fix?

Reply:
no but magic happened, 
I also reduced the amount of segments cause so many are not needed